.idea/misc.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this file here? All .idea folder should be included in .gitignore and should not appear on GitHub